server/controllers/stock/index.js

```diff
@@ -176,58 +176,43 @@ function normalMovement(document, params) {
 
 /**
  * @function depotMovement
- * @description movement between depots, there are two lines for IN and OUT
+ * @description movement between depots
  */
 function depotMovement(document, params) {
   let paramIn;
   let paramOut;
   let isWarehouse;
-
   const transaction = db.transaction();
   const parameters = params;
+  const isExit = parameters.isExit;
 
-  parameters.enity_uuid = parameters.enity_uuid ? db.bid(parameters.enity_uuid) : null;
+  let record;
+  
+  parameters.entity_uuid = parameters.entity_uuid ? db.bid(parameters.entity_uuid) : null;
 
   parameters.lots.forEach((lot) => {
-        // OUT:
-    paramOut = {
-      uuid          : db.bid(uuid.v4()),
-      lot_uuid      : db.bid(lot.uuid),
-      depot_uuid    : db.bid(parameters.from_depot),
-      document_uuid : db.bid(document.uuid),
-      quantity      : lot.quantity,
-      unit_cost     : lot.unit_cost,
-      date          : document.date,
-      entity_uuid   : null,
-      is_exit       : 1,
-      flux_id       : core.flux.TO_OTHER_DEPOT,
-      description   : parameters.description,
-      user_id       : document.user,
-    };
 
-        // IN:
-    paramIn = {
+    record = {
       uuid          : db.bid(uuid.v4()),
       lot_uuid      : db.bid(lot.uuid),
-      depot_uuid    : db.bid(parameters.to_depot),
+      depot_uuid    : isExit ? db.bid(parameters.from_depot) : db.bid(parameters.to_depot),
       document_uuid : db.bid(document.uuid),
       quantity      : lot.quantity,
       unit_cost     : lot.unit_cost,
       date          : document.date,
-      entity_uuid   : null,
-      is_exit       : 0,
-      flux_id       : core.flux.FROM_OTHER_DEPOT,
+      entity_uuid   : isExit ? db.bid(parameters.to_depot) : db.bid(parameters.from_depot),
```

Comment:
Instead of doing this on every turn of the loop, can you do something like this:
```js
const depot_uuid =  db.bid(isExit ? parameters.from_depot : parameters.to_depot);
const entity_uuid = db.bid(isExit ? parameters.to_depot : parameters.from_depot);
const is_exit = isExit ? 1 : 0;
const flux_id = isExit ? core.flux.TO_OTHER_DEPOT : core.flux.FROM_OTHER_DEPOT;

parameters.lots.forEach((lot) => {
    record = {
      depot_uuid,
      is_exit,
      entity_uuid,
      // ... etc ...
    };
});
```